chatterbot/adapters/logic/time_adapter.py

```diff
@@ -0,0 +1,32 @@
+from .logic import LogicAdapter
+from chatterbot.conversation import Statement
+from textblob.classifiers import NaiveBayesClassifier
+from datetime import datetime
+
+
+class TimeLogicAdapter(LogicAdapter):
+
+    def __init__(self, **kwargs):
+        super(TimeLogicAdapter, self).__init__(**kwargs)
+
+        training_data = [
+            ("what time is it", 1),
+            ("do you know the time", 1),
+            ("do you know what time it is", 1),
+            ("what is the time", 1),
+            ("do you know the time", 0),
+            ("it is time to go to sleep", 0),
+            ("what is your favorite color", 0),
+            ("i had a great time", 0)
+        ]
+
+        self.classifier = NaiveBayesClassifier(training_data)
```

Comment:
Ok, I agree with that. Let me give this some thought because I have been considering doing something similar with parts of the logic in the closest_meaning and closest_match adapters. I hand not been thinking in terms of classifiers so that may be a good approach to take. The only constraint or possible road block I can possibly see is the case that each classifier cannot be made generic enough to be usable in other existing or new adapters.

Again, I'll need to give this some thought and experiment a bit with the concept, but it might be a step in the right direction. I many end up opening a new ticket about this if I come up with a good solution.